development/PIP_INSTALL.md

```diff
@@ -0,0 +1,149 @@
+# Python PIP Install Instructions
+
+Pacifica is highly recommended to use virtual environments to install manually.
```

Comment:
This sentence structure doesn't quite read right.

Reply:
It also says environments (plural), what environments and how many are needed

Reply:
Added some more specific docs on this, the answer is both, you can use many or you can use one.